superset-frontend/src/dashboard/actions/hydrate.js

```diff
@@ -291,8 +292,26 @@ export const hydrateDashboard =
       future: [],
     };
 
+    // Searches for a focused_chart parameter in the URL to automatically focus a chart
+    const { focused_chart: focusedChartId } = regularUrlParams;
+    let focusedChartLayoutId;
+    if (focusedChartId) {
+      // Converts focused_chart to dashboard layout id
+      const found = Object.values(dashboardLayout.present).find(
+        // eslint-disable-next-line eqeqeq
```

Comment:
eqeqeq? 😄

Reply:
It made me laugh too! 🤣

Reply:
Why do we need to use `==`? Are meta.chartId and focusedChartId dofferent types? In that case we could try casting to number. Alternatively we could add focused_chart to URL_PARAMS as number type and retrieve it with getUrlParam - it would be cast to number automatically

Reply:
Done